srcpkgs/sq/template

```diff
@@ -0,0 +1,28 @@
+# Template file for 'sq'
+pkgname=sq
+version=0.40.0
+revision=1
+archs="x86_64* aarch64*"
```

Comment:
what's the reasoning to limit archs here?

Reply:
It doesn't build in 32-bit systems, at least [in CI](https://github.com/void-linux/void-packages/actions/runs/5641324418)

Reply:
why preclude ppc64  and riscv64 then?

Reply:
Void doesn't officially support ppc anymore, does it? And I have no way to verify if it actually builds for riscv so I didn't include it.

Reply:
yet there's no reason to spoil the fun for those who build on their own.